src/main/java/com/blackducksoftware/integration/fortify/batch/util/RestConnectionHelper.java

```diff
@@ -54,6 +57,8 @@
      * @return
      */
     private static HubServerConfig getHubServerConfig(PropertyConstants propertyConstants) {
+        logger.info("Inside settings::" + propertyConstants.getHubServerUrl() + ", " + propertyConstants.getHubUserName() + ", "
+                + propertyConstants.getHubPassword());
```

Comment:
Are we logging passwords here?

Reply:
I have added it while debugging, forget to remove it. Thanks for catching it.